modules/nw-dns-operator-managementState.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * networking/dns-operator.adoc
+
+[id="nw-dns-operator-managementState_{context}"]
+= Changing the DNS Operator managementState
+
+DNS manages the CoreDNS component to provide a name resolution service for pods and services in the cluster. The `managementState` of the DNS Operator is set to `Managed` by default, which means that the DNS Operator is actively managing its resources. You can change it to `Unmanaged`, which means the DNS Operator is not managing its resources.
+
+Use cases for changing the DNS Operator `managementState` include:
```

Comment:
Suggestion below based on IBM style guide for lead-in wording. It's a bit more straightforward for translation purposes. 

```suggestion
The following are use cases for changing the DNS Operator `managementState`:
```